app/tests/spec/models/auth_brokers/fx-desktop.js

```diff
@@ -116,6 +116,7 @@ define([
         return broker._notifyRelierOfLogin(account)
           .then(function () {
             assert.equal(data.email, 'testuser@testuser.com');
+            assert.isFalse(data.verified);
```

Comment:
I get it, I guess. You're making sure all tests of `_notifyRelierOfLogin` do the right thing.